packages/studio-base/src/panels/index.ts

```diff
@@ -38,21 +38,21 @@ const builtin: PanelInfo[] = [
   {
     title: `Diagnostics – Detail`,
     type: "DiagnosticStatusPanel",
-    description: "Display data for a given diagnostics node.",
+    description: "Display data for a given ROS diagnostics node.",
     help: DiagnosticStatusPanelHelp,
     module: async () => await import("./diagnostics/DiagnosticStatusPanel"),
   },
   {
     title: `Diagnostics – Summary`,
     type: "DiagnosticSummary",
-    description: "Display the status of diagnostics nodes.",
+    description: "Display the status of ROS diagnostics nodes.",
     help: DiagnosticSummaryHelp,
     module: async () => await import("./diagnostics/DiagnosticSummary"),
   },
   {
     title: "Image",
     type: "ImageViewPanel",
-    description: "Display camera feed images.",
+    description: "Display image topic messages.",
```

Comment:
the words "topic" and "messages" are not particularly useful or unique to this panel

```suggestion
    description: "Display compressed or uncompressed images.",
```